indico/modules/rb_new/controllers/backend/common.py

```diff
@@ -43,3 +43,27 @@
     'ne_lat': fields.Float(validate=lambda x: -90 <= x <= 90),
     'ne_lng': fields.Float(validate=lambda x: -180 <= x <= 180)
 }
+
+room_args = {
+    'building': fields.Int(required=True),
+    'capacity': fields.Int(validate=lambda x: 1 <= x, missing=20),
+    'comments': fields.Str(),
+    'division': fields.Str(),
+    'floor': fields.Int(required=True),
+    'full_name': fields.Str(),
+    'has_projector': fields.Bool(required=True),
+    'has_vc': fields.Bool(required=True),
+    'has_webcast_recoding': fields.Bool(required=True),
```

Comment:
typo (missing 'r' in 'recording'), and also a property so it cannot be changed (it's currently based on existence of a given equipment type)